src/wallet/test/coinselector_tests.cpp

```diff
@@ -210,6 +211,9 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
         const auto result9 = SelectCoinsBnB(GroupCoins(available_coins.All()), 1 * CENT, coin_selection_params_bnb.m_cost_of_change);
         BOOST_CHECK(result9);
         BOOST_CHECK_EQUAL(result9->GetSelectedValue(), 1 * CENT);
+        // Demonstrate how following improvements reduce iteration count and catch any regressions in the future.
```

Comment:
In df20dd4f905a0d04e48bfe20919cccaff11b9b30 "coinselection: Track BnB iteration count in result"

This kind of comment is belongs in the commit message, not the code.